jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/service/ResourceApiService.java

```diff
@@ -73,14 +74,27 @@ public T updateResource(T entity) {
     });
   }
 
-  public List<T> updateResources(List<T> entities) {
+  public List<T> updateResources(List<T> entities, UUID tenantId) {
 
     return transactionTemplate.execute(status -> {
+      var ids = new ArrayList<UUID>();
```

Comment:
I don't think we need to store the IDs separately. 
If you explicitly type the HashMap below with `<UUID, T>` then you should just be able to pass the set of id keys to the repo call.

Reply:
so you shouldn't need to instantiate a new `List` or anything else, you should be able to pass in the `Set` as is.
Does that work?

Reply:
Fixed, I though the test was failing because of the type of the keySet, but it was a problem with the mock in the test